email.go

```diff
@@ -110,6 +111,23 @@ func (encryption Encryption) String() string {
 	return encryptionTypes[encryption]
 }
 
+type headerEncoding int
+
+const (
+	// HeaderEncodingNone turns off encoding on the message headers
+	// https://www.rfc-editor.org/rfc/rfc6530#section-7.1
+	HeaderEncodingNone headerEncoding = iota
+
+	// TODO: Add Base64 encoding
+	// HeaderEncodingBase64 sets the message header encoding to base64
+	// https://www.rfc-editor.org/rfc/rfc2045#section-6.8
+	// HeaderEncodingBase64
```

Comment:
Left a TODO for now. I don't personally have a use for this, but if I do get some time, I'll try to add it in.